api/src/main/java/io/strimzi/kafka/access/model/KafkaReference.java

```diff
@@ -18,8 +18,10 @@
 public class KafkaReference {
 
     @Required
+    @io.fabric8.crd.generator.annotation.PrinterColumn(name = "Cluster")
     private String name;
     private String namespace;
+    @io.fabric8.crd.generator.annotation.PrinterColumn()
```

Comment:
Should we have some better name here as well such as "Listener"? The CRDs look a bit  strange with `User`, `Cluster` and `LISTENER`.

Reply:
Done